### ISSUE TITLE
Windows 7 GUI doesn't work via sys-gui-gpu

### ISSUE THREAD

**Issue by ZYousef** [QubesOS/qubes-issues#7565]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

Qubes R4.1

### Brief summary

sys-gui-gpu cannot display windows 7 - qga-enabled qube.

# ## Steps to reproduce

setup sys-gui-gpu qube
setup windows 7 qube
try to run windows qube inside sys-gui-gpu desktop environment 

### Expected behavior

windows 7 applications launch normally in the new desktop environment.

### Actual behavior

windows 7 launches applications without GUI.

**Comment by marmarek** [comment]
Yes, this features is not implemented in windows gui agent. I think we had an issue for tracking this already, but can't find it now, @andrewdavidwong ?

**Comment by andrewdavidwong** [comment]
> Yes, this features is not implemented in windows gui agent. I think we had an issue for tracking this already, but can't find it now, @andrewdavidwong ?

Is it #4886?

**Comment by ZYousef** [comment]
are there plans to support this in the new QWT 4.2.0 for QR4.3 ? @omeg 
https://github.com/omeg/qubes-installer-windows-tools

**Comment by omeg** [comment]
The new Windows 10/11 gui agent does use grant-based framebuffer sharing (support for this was added to upstream Windows pvdrivers by me a while ago).

**Comment by ZYousef** [comment]
I tried it and it works in adminvm seamlessly but when I switched to sys-gui it does no longer work.

The GUI daemon doesn't complain about anything I just see 'Starting GUI: windows-vm' in xsession-errors.log

How to reproduce:
Qubes OS R4.3
- Used qvm-create-windows-qube tool
With windows10 iso.
- Run windows-qube seamlessly in dom0: works
- Turn off windows-qube
- qvm-prefs windows-qube guivm sys-gui
- Start sys-gui then start windows-qube: it doesn't work.

**Comment by marmarek** [comment]
That's done in https://github.com/QubesOS/qubes-gui-agent-windows/pull/7
